Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/General/Scripts/TargetBehaviors/LoadAdditiveScene.cs

```diff
@@ -26,45 +26,23 @@ public class LoadAdditiveScene : MonoBehaviour
         [Tooltip("Timeout in seconds before new scene is loaded.")]
         private float waitTimeInSecBeforeLoading = 0.25f;
 
-
         public void LoadScene()
         {
             LoadScene(SceneToBeLoaded);
         }
 
-        public void LoadScene(string sceneName)
+        public async void LoadScene(string sceneName)
```

Comment:
Given that the script name is "LoadAdditiveScene", it might be nice to either use a completely new script and leave this one as it is. Or to also rename the script itself.